src/Akeneo/Crowdin/Api/UploadTranslation.php

```diff
@@ -92,25 +93,26 @@ public function execute()
     }
 
     /**
-     * @param string $crowdinPath the Crowdin file path
-     * @param string $localPath   the local file path
+     * @param string $localPath
+     * @param string $crowdinPath
+     * @param string $exportPattern
+     * @param string $title
      *
-     * @throws InvalidArgumentException
-     *
-     * @return UploadTranslation
+     * @return $this
      */
-    public function addTranslation($crowdinPath, $localPath)
+    public function addTranslation($localPath, $crowdinPath, $exportPattern = null, $title = null)
```

Comment:
Did you just switch the 2 arguments? Am I missing something or is it a massive BC break?